### PR TITLE
Add form header text and two input fields

### PR DESCRIPTION
**Pull request by fahim-86**
This pull request contains below section of the Form:

- Header text
- Body text
- Two input fields

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -179,6 +182,25 @@ <h1>About <br />Myself</h1>
           </div>
         </div>
       </section>
+      <section class="pf-contact-form">
+        <div class="form-div">
```
Comment (RaoAkif) [inline index.html:186]: You have done very well in designing this section. 
You have put the right background image in `form-div` class, but it is a missing few elements from the bottom. Kindly adjust the image to include full background with the current styling.
This is Figma Design:
image

This is your design:
image

File: index.html [hunk 2]
```diff
@@ -179,6 +182,25 @@ <h1>About <br />Myself</h1>
...
+          <div class="cf-form">
+            <form action="https://formspree.io/f/xrgvblrz" method="POST">
+              <input type="text" id="fname" name="fname" placeholder="Input your name here" maxlength="30" required />
+              <input type="email" id="email" name="email" placeholder="Input your email here" required />
+              <textarea id="message" maxlength="500" placeholder="Write your message here:" required></textarea>
+              <button type="submit" class="submit-button">Get In Touch</button>
+            </form>
+          </div>
+        </div>
+      </section>
```
Comment (RaoAkif) [inline index.html:203]: The following (horizontal line) shape is missing in your UI design. Kindly add it to match the provided FIGMA design.
image

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @fahim-86, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- There are few colors in your `styles.css` for example `#6070ff` and `#fff`, which are being used over and over again. To write DRY code, you can create a variable for each of them, and use them everywhere. In this way, If you would like to change the theme color, you would have to change it at one position, and you would be done.
For Example:
```
  --color-primary: #272a31;
  --color-secondary: #ec5242;
  --color-tertiary: #d3d3d3;
  --color-white: #fff;
  --color-grey: #585858;
  --color-light-grey: #ebf0ee;
```
or like
```
  --primary: #272a31;
  --secondary: #ec5242;
  --tertiary: #d3d3d3;
```


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi  @fahim-86 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good Design 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**